specification/simulate/ingest/SimulateIngestResponse.ts

```diff
@@ -17,8 +17,69 @@
  * under the License.
  */
 
-import { SimulateDocumentResult } from '@ingest/_types/Simulation'
+import { AdditionalProperties } from '@spec_utils/behaviors'
+import { Dictionary } from '@spec_utils/Dictionary'
+import { Stringified } from '@spec_utils/Stringified'
+import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
+import { Id, IndexName, VersionNumber } from '@_types/common'
+import { ErrorCause } from '@_types/Errors'
 
 export class Response {
-  body: { docs: SimulateDocumentResult[] }
+  body: { docs: SimulateIngestDocumentResult[] }
+}
+
+export class SimulateIngestDocumentResult {
+  doc?: IngestDocumentSimulation
+}
+
+/**
+ * The results of ingest simulation on a single document. The _source of the document contains
+ * the results after running all pipelines listed in executed_pipelines on the document. The
+ * list of executed pipelines is derived from the pipelines that would be executed if this
+ * document had been ingested into _index.
+ *
+ * @behavior_meta AdditionalProperties fieldname=metadata description="Additional metadata"
+ */
+export class IngestDocumentSimulation
+  implements AdditionalProperties<string, string>
+{
+  /**
+   * Identifier for the document.
+   */
+  _id: Id
+  /**
+   * Name of the index that the document would be indexed into if this were not a simulation.
+   */
+  _index: IndexName
+  /**
+   * JSON body for the document.
+   */
+  _source: Dictionary<string, UserDefinedValue>
+  /**
+   *
+   */
+  _version: Stringified<VersionNumber>
+  /**
+   * A list of the names of the pipelines executed on this document.
+   */
+  executed_pipelines: Array<string>
+  /**
+   * A list of the fields that would be ignored at the indexing step. For example, a field whose
+   * value is larger than the allowed limit would make it through all of the pipoelines, but
+   * would not be indexed into Elasticsearch.
+   */
+  ignored_fields?: Array<Dictionary<IgnoredFieldKey, string>>
```

Comment:
this dictionary is always just one "field" + string right? we have the type SingleKeyDictionary if so
```suggestion
  ignored_fields?: Array<SingleKeyDictionary<IgnoredFieldKey, string>>
```

Reply:
let's leave it like this then